crates/hyperswitch_connectors/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "hyperswitch_connectors"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+
+[dependencies]
+async-trait = "0.1.79"
+error-stack = "0.4.1"
+serde = { version = "1.0.197", features = ["derive"] }
+serde_json = "1.0.115"
+
+# First party crates
+api_models = { version = "0.1.0", path = "../api_models", features = ["errors"] }
+cards = { version = "0.1.0", path = "../cards" }
+common_enums = { version = "0.1.0", path = "../common_enums" }
+common_utils = { version = "0.1.0", path = "../common_utils", features = ["signals", "async_ext", "logs", "metrics"] }
+diesel_models = { version = "0.1.0", path = "../diesel_models", features = ["kv_store"] }
```

Comment:
what are the dependency connected to diesel_models?

Reply:
The dependency was not being used directly and is now removed.